### ISSUE TITLE
Change Request: Keep the Cancel Button in a Stable location

### ISSUE THREAD

**Issue by Copyrites** [theotherp/nzbhydra2#295]
## How to request a new feature
* What do you want to happen?

Move the "cancel" button to the top of the status box so its position is stable.

* If not clear, why do you want it?

Currently the cancel button moves down the screen as each indexer is searched.  That requires the user to "chase" the button as it moves, which makes cancelling unnecessarily difficult.

* Do you think it's something only you need or something that might be popular?

It affects everyone who wants to cancel a search in progress.

**Comment by albino1** [comment]
+1

**Comment by theotherp** [comment]
You guys are just too slow... :-)

Seriously, that's kinda an obvious idea... By the way, I *think* you can just press Escape. Not sure (you'd think I should know that).

**Comment by albino1** [comment]
Yeah, I use escape, but sometimes I forget and just instinctively start chasing that darn Cancel button :)

**Comment by albino1** [comment]
Thanks!

**Comment by biship** [comment]
Lol, thanks!
